planner/core/cbo_test.go

```diff
@@ -1001,19 +1001,19 @@ func (s *testAnalyzeSuite) TestLimitCrossEstimation(c *C) {
 	tk.MustExec("set session tidb_opt_correlation_exp_factor = 0")
 	// Pseudo stats.
 	tk.MustQuery("EXPLAIN SELECT * FROM t WHERE b = 2 ORDER BY a limit 1;").Check(testkit.Rows(
-		"TopN_8 1.00 root test.t.a:asc, offset:0, count:1",
-		"└─IndexReader_16 1.00 root index:TopN_15",
-		"  └─TopN_15 1.00 cop test.t.a:asc, offset:0, count:1",
-		"    └─IndexScan_14 10.00 cop table:t, index:b, range:[2,2], keep order:false, stats:pseudo",
+		"Limit_11 1.00 root offset:0, count:1",
```

Comment:
@eurekaka I think that these changes breaks what we want to test?

Reply:
How about changing the DDL to:
```sql
create table t(a int primary key, b int not null, c int not null, index idx_b_c(b,c))
```